include/zenoh/api/session.hxx

```diff
@@ -48,10 +48,11 @@ class Session : public Owned<::z_owned_session_t> {
     struct SessionOptions {
         /// @name Fields
 #ifdef ZENOHCXX_ZENOHPICO
-        /// @brief For zenoh-pico only. If true, start background threads which handles the network
-        /// traffic. If false, the threads should be called manually with ``Session::start_read_task`` and
+        /// @brief If ``true``, start background thread which handles the network
```

Comment:
as far as I remember, there are two background threads: read_task and lease_task
And why "For zenoh-pico only" comment is removed

Reply:
it is not removed, it is moved to @note below.

Reply:
Fixed back thread to threads.